pkg/chains/address.go

```diff
@@ -68,19 +69,14 @@ func DecodeBtcAddress(inputAddress string, chainID int64) (address btcutil.Addre
 	if chainParams == nil {
 		return nil, fmt.Errorf("chain params not found")
 	}
-	// test taproot address type
-	address, err = DecodeTaprootAddress(inputAddress)
-	if err == nil {
-		if address.IsForNet(chainParams) {
-			return address, nil
-		}
-		return nil, fmt.Errorf("address %s is not for network %s", inputAddress, chainParams.Name)
-	}
-	// test taproot address failed; continue testing other types: P2WSH, P2WPKH, P2SH, P2PKH
+
+	// try decoding input address as a Bitcoin address
```

Comment:
Maybe mention the address types that work now with the decode ?
P2WSH, P2WPKH, P2SH, P2PKH, TapRoot